### ISSUE TITLE
developing a website layout and branding

### ISSUE THREAD

**Issue by tobygeorge** [OREL-group/Project-Management-FA23#134]

**Comment by tobygeorge** [comment]
In order to communciate what our company seeks to achieve, I recommend that we have a website that effectively details what people can do with our company. Moreover, our branding needs to communicate our values and missions. Moreover the more aesethetic our page the more likely people are to engage with our company.

**Comment by bryange3** [comment]
I agree, let's consult with our marketing & branding team next week.
